desktop/core/src/desktop/lib/raz/raz_client.py

```diff
@@ -215,6 +220,7 @@ def handle_adls_req_mapping(self, method, params, relative_path):
         relative_path += lib_urlunquote(params['directory'])
     
     if method == 'PUT':
+      access_type = 'rename-destination'
```

Comment:
Probably need to add the headers in the params of handle_adls_req_mapping() to check for this case, then easier to add a test

Reply:
And to also sign the `x-ms-rename-source` path